utilities/message_filters/src/message_filters/__init__.py

```diff
@@ -87,10 +87,13 @@ class Cache(SimpleFilter):
 
     Given a stream of messages, the most recent ``cache_size`` messages
     are cached in a ring buffer, from which time intervals of the cache
-    can then be retrieved by the client.
+    can then be retrieved by the client. The ``allow_headerless``
+    option specifies whether to allow storing headerless messages with
+    current ROS time instead of timestamp. You should avoid this as
+    much as you can, since the delays are unpredictable.
     """
 
-    def __init__(self, f, cache_size = 1):
+    def __init__(self, f, cache_size = 1, allow_headerless = False):
```

Comment:
Can you please remove the spaces around the `=` sign (following PEP 8) (also for the other keyword).